test/nql_knex.test.js

```diff
@@ -12,254 +12,13 @@ const makeQuery = (nqlString, options) => {
     const postKnex = knex('posts');
     const filter = nql.parse(nqlString, options);
 
-    // console.log('Filter', require('util').inspect(filter, false, null)); // eslint-disable-line no-console
-
     return mongoToKnex(postKnex, filter, {});
```

Comment:
wondering why the integration tests call the sub dependency manually?
instead of using the API of NQL?

Reply:
Looked into it a bit more and the main api suite covers more than this test at the moment - https://github.com/NexesJS/NQL/blob/master/test/api.test.js#L35. Think we'd need something similar integration tests in mongo-knex here to check the base case, and the test would then look more like:
```
    it('should match based on simple id', () => {
        // set up the db first

        const query = nql('id:3');

        return query
            .querySQL(knex('posts'))
            .select()
            .then((result) => {
                result.should.be.an.Array().with.lengthOf(3);
            });
    });
```
Should I go on and implement this ^ ? What are your thought @kirrg001 ?

Reply:
👍